test/fixtures/readProjectFile.js

```diff
@@ -8,5 +8,9 @@ module.exports = {
     extractProjectJson: function (path) {
         const zip = new AdmZip(path);
         return JSON.parse(zip.readAsText('project.json', 'utf8'));
+    },
+    extractNestedProjectJson: function (path, folder) {
```

Comment:
I think it would be better for this logic to go inside `extractProjectJson` so that we don't have to think about how the fixture was created when writing a test.

I think instead of taking the folder name, it should do something similar to the code changes in this PR and in scratch-parser where it looks for the `project.json` in a folder if it fails to find it at the top level.

There's an API function listed here (`getEntryChildren`) that might be helpful:

https://github.com/cthackers/adm-zip/wiki/ZipFile

Reply:
I used `getEntries` instead so that it didn't need to know the folder name.